spyder/plugins/editor/plugin.py

```diff
@@ -287,27 +287,29 @@ def restore_scrollbar_position(self):
     def report_open_file(self, options):
         """Request to start a LSP server to attend a language."""
         filename = options['filename']
-        logger.debug('Call LSP for %s' % filename)
         language = options['language']
+        logger.debug('Call LSP for %s [%s]' % (filename, language))
         callback = options['codeeditor']

```

Comment:
Let's change the name of this variable to `codeeditor` because it's no longer a `callback`.